can_tools/scrapers/official/base.py

```diff
@@ -2,11 +2,14 @@
 from abc import ABC
```

Comment:
We typically order the imports in the following way:

* Break imports into 3 blocks: (1) standard lib imports, (2) external package imports, (3) imports from your own code/library
* Each import block first does all of the `import X as x` then `from X import x`

`json` and `re` are part of the standard lib.

See https://www.python.org/dev/peps/pep-0008/#imports